backend/app/consts.py

```diff
@@ -0,0 +1,48 @@
+"""Common routes constants"""
+from typing import Literal, TypeAlias
+
+# Collection names in Solr and ess
+Collection: TypeAlias = Literal[
+    "all_collection",
+    "bundle",
+    "data_source",
+    "dataset",
+    "guideline",
+    "other_rp",
+    "publication",
+    "service",
+    "training",
+    "provider",
+    "software",
+]
+
+ResearchProductCollection: TypeAlias = Literal[
+    "publication", "dataset", "software", "other_rp"
+]
+
+# Recommender collection names
+PanelId: TypeAlias = Literal[
+    "data_sources",
+    "datasets",
+    "other_research_product",
+    "publications",
+    "services",
+    "trainings",
+    "software",
+]
+
+COLLECTION_TO_PANEL_ID_MAP = {
+    "all_collection": "all",
+    "data_source": "data-sources",
+    "dataset": "datasets",
+    "other_rp": "other_research_product",
+    "publication": "publications",
+    "service": "services",
+    "training": "trainings",
+    "software": "software",
+}
+
+PROVIDER_QF = "title^100 description^10 scientific_domains^10"
```

Comment:
I would also consider creating two separate files. One for collection names, second for solr-specific stuff.